### ISSUE TITLE
[css-text] For most languages, hyphens:auto should not hyphenate Capitalized words

### ISSUE THREAD

**Issue by mozilla-apprentice** [heycam/csswg-decisions#778]
A resolution was made for [csswg-drafts/#3927](https://github.com/w3c/csswg-drafts/issues/3927).

**\[css\-text\] For most languages, hyphens:auto should not hyphenate Capitalized words**

* RESOLVED: Add A note to the spec and close with no normative change

[Discussion.](https://github.com/w3c/csswg-drafts/issues/3927#issuecomment-1107443038)

----

To file a bug automatically for these resolutions, add the **bug** label to the issue.

If no bug is needed, the issue can be closed.

**Comment by Atnas-dev** [comment]
Due to language rules, some browsers has decided that automatic hyphenation is not allowed on capitalised words as these are seen as "Proper Nouns".

This is indeed true and respects correct gramma within the designated language.
However; it creates the bug within the browsers themselves that the first word of a sentence (if capitalised) will NEVER be hyphenated.
Also if an editor decides to write a headline in capitalised style, corresponding to the correct language rules of capitalisation. The allowed and truthful capitalised words within this heading will never be capitalised.

It is essential for supporting of creative, appealing, content rich and language appropriate hyphenation.
Stating that english will NEVER hyphenate a capitalised word is simply wrong. Stating that English will never hyphenate a proper noun is very true. But the detection of a proper noun being by the style of the text seems to be a very vague and incorrect indication.

In other words. If this issue can not be handled correctly, maybe it should not have been handled at all in the first place?

As mentioned in many other discussions about this issue, an additional hyphenation rule could be added to define whether or not capitalised words should be hyphenated in the given element.
